### ISSUE TITLE
Add APT repository

### ISSUE THREAD

**Issue by silopolis** [rustdesk/rustdesk#951]
Hi,

This great and so much needed piece of code deserves an APT repository to ease install and updates on Debian derivatives !

Thanks for all

TY
J

**Comment by simeononsecurity** [comment]
@silopolis I agree this would be nice, however if they did this, they would also have to implement repos for the other various OS'es. This would unfortunately require a fair bit of work to implement and eventually automate. However, I agree this should definitely happen at some point. However with the software in what I'd consider to be still beta, maybe even alpha, it's not the highest thing on the list.

**Comment by Oceanwatcher** [comment]
I second this. A Ubuntu repository would first of all make it really easy to install. Just do a sudo apt install rustdesk.

But it would also make it real easy to keep updated as you can set the system to do minor updates automatically. This improves the security and should count for a lot. Actually, the upgrade process today is not documented as far as I can see. It should be as easy as sudo apt update and sudo apt upgrade ;-)

I avoid docker as the plague and prefer to have a small VPS for the server instead. And these days you can set up a small VPS on Oracle for free. It will at least help with the testing.

And I completely disagree that it has do be done for all other OS'es as well. At least not from the start.Ubuntu is a solid, widely used OS and would be a natural place to start. If other OS'es has similar delivery methods, then yes, it should be natural to add them eventually. But at least getting one OS up and running with this will lower the threshold for people to set up their own relay server.

**Comment by hex-m** [comment]
The most sustainable way to solve this would be to add rustdesk to the official repository of Debian. Ubuntu automatically gets packages that are in Debian ([src](https://wiki.ubuntu.com/UbuntuDevelopment/NewPackages)).

> Packages that have recently been added to Debian unstable will be automatically synced into Ubuntu

So the next step would be a [Debian RFP](https://wiki.debian.org/RFP).

**Comment by lweberk** [comment]
In the meantime I've been distributing it over a private repository, for which the problem becomes the versioning. Development versions (nightly) do not tag the dev portion (pre-release for ex.) in their release schema. It makes it really hard to propagate new versions this way, since there is nothing telling the packaging managers about the new nightly actually being newer than the one the day before.

**Comment by simeononsecurity** [comment]
It's easy, maintain a separate repo for nightly.
If you're going to install a nightly, potentially buggy release, it should be specific and determined by the users. 
The base install, that should be going to debian should be somewhat stable...

**Comment by Jackenmen** [comment]
> In the meantime I've been distributing it over a private repository, for which the problem becomes the versioning. Development versions (nightly) do not tag the dev portion (pre-release for ex.) in their release schema. It makes it really hard to propagate new versions this way, since there is nothing telling the packaging managers about the new nightly actually being newer than the one the day before.

I do this as well and I just reversion those deb files with a release counter, i.e. my current rustdesk version is `1.2.0-0gemfury64`. `deb-reversion` could be used, if not for the fact that rustdesk's deb file doesn't actually contain [the required `debian/changelog` file](https://www.debian.org/doc/manuals/maint-guide/dreq.en.html#changelog). If that file were there, it's simply a matter of running something like `fakeroot deb-reversion -v 1.2.0-0jackenmen1 rustdesk.deb`. Personally, I just decided to [reimplement what `deb-reversion` does in Python](https://github.com/Jackenmen/sync-debs-to-gemfury/blob/main/src/sync_debs_to_gemfury/deb_reversion.py) with some slight modifications and use that instead but asking for a `debian/changelog` file in another issue is also an option :)
